frontend/src/App.css

```diff
@@ -26,3 +26,6 @@ html {
   text-align: center;
 }
 
+.footer > * {
+  margin: 10px;
```

Comment:
It leads all element in footer will have this margin in every direction, do you want it? I suggest that you can set `className` for the element you want to add style to them

Reply:
Currently, this is what I want, because all the 3 elements need to be margined with each other. A class name is redundant for them before we have more complicated layouts in the footer.